sqlx-postgres/src/options/mod.rs

```diff
@@ -683,6 +692,12 @@ fn test_options_formatting() {
         options.options,
         Some("-c geqo=off -c statement_timeout=5min".to_string())
     );
+    // https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-OPTIONS
+    let options = PgConnectOptions::new().options([("passfile", "/back\\slash/ and\\ spaces")]);
+    assert_eq!(
+        options.options,
+        Some("-c passfile=/back\\\\slash/\\ and\\\\\\ spaces".to_string())
```

Comment:
This can be made a lot nicer with [raw string literals](https://doc.rust-lang.org/reference/tokens.html#raw-string-literals):

```suggestion
    let options = PgConnectOptions::new().options([("passfile", r"/back\slash/ and\ spaces")]);
    assert_eq!(
        options.options,
        Some(r"-c passfile=/back\\slash/\ and\\\ spaces".to_string())
```